src/components/ContextMenu/index.js

```diff
@@ -70,9 +70,9 @@ const enhance = compose(
       event.preventDefault();
       const { CmdHandler } = pttchrome;
       const doDOMMouseScroll =
-        CmdHandler.getAttribute("doDOMMouseScroll") === "1";
+        CmdHandler.getAttribute("USING_doDOMMouseScroll") === "1";
```

Comment:
Why are these all prefixed with `USING_`?

Reply:
I have to prefix some of them with `USING_` so that I can identify unused ones. Should I rename them back?

Reply:
Yes, please do. (since the purpose of the rename is just for tracing code)